wp-content/plugins/vf-ebi-global-footer-container/index.php

```diff
@@ -132,7 +132,7 @@ public function wp_enqueue_scripts() {
     );
     wp_enqueue_style(
       'ebi-fonts',
-      'https://dev.ebi.emblstatic.net/web_guidelines/EBI-Icon-fonts/v1.3/fonts.css',
+      'https://ebi.emblstatic.net/web_guidelines/EBI-Icon-fonts/v1.3/fonts.css',
```

Comment:
Is it fine to keep fonts 1.3? or they also need to update to 1.4

Reply:
There's actually no fonts v1.4, it's a separate project version.